prov/util/src/util_srx.c

```diff
@@ -496,6 +501,33 @@ static struct util_rx_entry *util_search_unexp_msg(struct util_srx_ctx *srx,
 	return util_search_peer_msg(ofi_array_at(&srx->src_unexp_peers, addr));
 }
 
+static bool util_unexp_mrecv(struct util_srx_ctx *srx,
+			     struct util_rx_entry *mrecv_entry,
+			     struct util_rx_entry *rx_entry)
+{
+	mrecv_entry->multi_recv_ref++;
+	rx_entry->peer_entry.owner_context = mrecv_entry;
+
+	rx_entry->peer_entry.iov[0].iov_base =
+			mrecv_entry->peer_entry.iov->iov_base;
+	rx_entry->peer_entry.iov->iov_len =
```

Comment:
It's a bit strange that we use `iov[0]` for iov_base while `iov->` for iov_len. Is it enforced that multi_recv post has to be 1 count?

Reply:
Yes, current util code only supports iov_count == 1 for multi_recv. See the assertion in `util_generic_mrecv`：
```
	assert(flags & FI_MULTI_RECV && iov_count == 1);
```